src/pybind/mgr/volumes/fs/operations/snapshot_util.py

```diff
@@ -19,13 +19,15 @@ def mksnap(fs, snappath):
         raise VolumeException(-e.args[0], e.args[1])
 
         
-def rmsnap(fs, snappath):
+def rmsnap(self, snapname, snappath):
```

Comment:
"self" is not a good parameter name. It is used for the current instance of a class. Please use something
 else. The parameter accepts SubvolumeBase objects. Maybe name it that hints to that, e.g., subvolumebaseobj, or something like that. Also describe the parameters in the docstring in reST format, https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats
 See the methods of SubvolumeTemplate class for docstring examples.

Reply:
Also this function is being called by Group's `remove_snapshot`, so it would be receiving `Group` class well. The Group's class doesn't have metadata_mgr.  Though the subvolumegroup snapshot creation is not allowed now. The subvolumegroup snapshot listing, rm is still supported for the existing snapshots. This would fail subvolumegroup snapshot rm.

Reply:
Undo these changes.

Reply:
Undo these changes